framework/src/problems/FEProblemBase.C

```diff
@@ -221,6 +204,14 @@ FEProblemBase::validParams()
            "be used (again, using the parameter '" +
            list_param_name + "').";
   };
+
+  params.addParam<std::vector<SubdomainName>>(
+      "block",
+      {},
```

Comment:
Can we remove the default and then just use `isParamValid`? To me an empty default (without knowing the checks that happen later with `isParamSetByUser`) suggests that by default we're not going to check coverage on any blocks

Reply:
```suggestion
```